### PR TITLE
Fix updating of user profile

### PR DESCRIPTION
**Pull request by jules-n**
Response handling changed

### CODE REVIEW

**Review by ChernykhVyacheslav — APPROVED** [review]
Looks good

File: src/components/ProfilePage/ProfilePage.js [hunk 1]
```diff
@@ -1,26 +1,31 @@
...
     const ref_lastName = useRef(null);
-    const ref_newPassword = useRef(null);
     const user = JSON.parse(sessionStorage.getItem('user'));
-    
+
+    const fieldHandling = (field, value) => {
+        if (field != null)
+            return (value.length === 0 || user[field] === value) ? null : value;
+        else return value.length === 0 ? null : value;
+    }
```
Comment (Dchad99) [inline src/components/ProfilePage/ProfilePage.js:19]: fieldHandling = (field, value) => {
     retrun value.length === 0 ? null : value 
}

What is the purpose to check the field? 
Why password property is null in update_user?

**Review by Dchad99 — CHANGES_REQUESTED** [review]

File: src/components/ProfilePage/ProfilePage.js (hunk 1 above)
Comment (jules-n) [reply src/components/ProfilePage/ProfilePage.js:19]: The user can change the password or leave old

**Review by nelfan — APPROVED** [review]
looks good

**Review by ChernykhVyacheslav — APPROVED** [review]

**Pull request merged**